docs/using_an_external_type.md

```diff
@@ -1,148 +1,4 @@
-
-Example external API Aggregation directory structure
-```
-github.com
-    ├── theiruser
-        ├── theirproject
-            ├── apis
-                ├── theirgroup
-                   ├── doc.go
-                   ├── install
-                   │   ├── install.go
-                   ├── v1alpha1
-                   │   ├── doc.go
-                   │   ├── register.go
-                   │   ├── types.go
-                   │   ├── zz_generated.deepcopy.go
-```
-
-In the case above the import path would be `github.com/theiruser/theirproject/apis/theirgroup/v1alpha1`
-
-### Create a project
-
-```
-kubebuilder init --domain $APIDOMAIN --owner "MyCompany"
-```
-
-### Add a controller
-
-be sure to answer no when it asks if you would like to create an api? [Y/n]
-```
-kubebuilder create api --group mygroup --version $APIVERSION --kind MyKind
-
-```
-
-## Edit the API files.
-
-### Register your Types
-
-Edit the following file to the pkg/apis directory to append their `AddToScheme` to your `AddToSchemes`:
-
-file: pkg/apis/mytype_addtoscheme.go
-```
-package apis
-
-import (
-	mygroupv1alpha1 "github.com/myuser/myrepo/apis/mygroup/v1alpha1"
-	theirgroupv1alpha1 "github.com/theiruser/theirproject/apis/theirgroup/v1alpha1"
-)
-
-func init() {
-	// Register the types with the Scheme so the components can map objects 
-	// to GroupVersionKinds and back
-	AddToSchemes = append(
-	  AddToSchemes, 
-	  mygroupv1alpha1.SchemeBuilder.AddToScheme,
-	  theirgroupv1alpha1.SchemeBuilder.AddToScheme,
-	)
-}
-
-```
-
-## Edit the Controller files
-
-### Use the correct imports for your API
-
-file: pkg/controllers/mytype_controller.go
-```
-import (
-	mygroupv1alpha1 "github.com/myuser/myrepo/apis/mygroup/v1alpha1"
-	theirgroupv1alpha1 "github.com/theiruser/theirproject/apis/theirgroup/v1alpha1"
-)
-```
-
-### Update dependencies
-
-```
-dep ensure --add
-```
-
-## Prepare for testing
-
-#### Register your resource
-
-Edit the `CRDDirectoryPaths` in your test suite by appending the path to their CRDs:
-
-file pkg/controllers/my_kind_controller_suite_test.go
-```
-var cfg *rest.Config
-
-func TestMain(m *testing.M) {
-	// Get a config to talk to the apiserver
-	t := &envtest.Environment{
-		Config:             cfg,
-		CRDDirectoryPaths:  []string{
-		  filepath.Join("..", "..", "..", "config", "crds"),
-		  filepath.Join("..", "..", "..", "vendor", "github.com", "theiruser", "theirproject", "config", "crds"),
-        },
-		UseExistingCluster: true,
-	}
-
-	apis.AddToScheme(scheme.Scheme)
-
-	var err error
-	if cfg, err = t.Start(); err != nil {
-		log.Fatal(err)
-	}
-
-	code := m.Run()
-	t.Stop()
-	os.Exit(code)
-}
-
-```
-
-## Helpful Tips
-
-### Locate your domain and group variables
-
-The following kubectl commands may be useful
-
-```
-kubectl api-resources --verbs=list -o name
-
-kubectl api-resources --verbs=list -o name | grep mydomain.com
-```
+This guide got moved and can now be found under `docs/reference/using_an_external_type.md`.
 
```

Comment:
We can remove the doc from here.
We do not to keep it because the same content is in the docs/website.

Reply:
Originally I wanted to remove the doc but I think many people are used to this location which is why I was only intending to keep a reference to the new doc here. I can remove it as well. 👍

Reply:
We have tags, branches and etc to be able to check what changed from older to new versions
So, IMO the best is remove since we are moving the content to the docs

Reply:
Removed this redirection file